build-system/tasks/cherry-pick.js

```diff
@@ -0,0 +1,173 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const argv = require('minimist')(process.argv.slice(2));
+const log = require('fancy-log');
+const {getOutput} = require('../common/exec');
+const {green, cyan, red, yellow} = require('ansi-colors');
+
+/**
+ * Executes a shell command, and logs an error message if the command fails.
+ *
+ * @param {string} cmd
+ * @param {string} msg
+ * @return {!Object}
+ */
+function execOrLog(cmd, msg) {
+  const result = getOutput(cmd);
+  if (result.status) {
+    log(yellow('ERROR:'), msg);
+    throw new Error(result.stderr);
+  }
+
+  return result;
+}
+
+/**
+ * Determines the name of the cherry-pick branch.
+ *
+ * @param {string} version
+ * @return {string}
+ */
+function cherryPickBranchName(version) {
+  const timestamp = version.slice(0, -3);
+  let suffix = String(Number(version.slice(-3)) + 1);
+  while (suffix.length < 3) {
+    suffix = `0${suffix}`;
+  }
```

Comment:
```js
const suffix = String(Number(version.slice(-3)) + 1).padStart(3, '0');
```

Reply:
Good to know this exists! I'll include it in a later PR